content/integrations/bitbucket.md

```diff
@@ -26,21 +26,28 @@ If you view a dashboard, in the top left search bar you can type ```sources:bitb
 
 ## Installation
 
-On the Bitbucket website, create a new user that has **Read** access to the repo you want to monitor. This is the user that Datadog will use to collect information on the repository.
+Setup your webhooks on the bitbucket side. [Instructions here][3].
+Use as the webhook url.:
+
+ ```
+ https://app.datadoghq.com/intake/webhook/bitbucket?api_key=YOUR_API_KEY
+ ```
 
 ## Configuration
 
-1.  Login to the Datadog application and add the [Bitbucket integration here][2].
-2.  Enter the username and password you created above in the Installation section. Click the **Install Integration** button. If the username or password is not correct you will get a error message.
-3.  Open the Bitbucket integration again and enter the full name of the repository you want to monitor. If the url for your repository is https://bitbucket.org/groupname/reponame, then enter ```groupname/reponame``` in the Repo full name textbox.
-4.  Decide what type of events you would like to collect and click **Update Configuration**.
+1. Login to the Datadog application and add the [Bitbucket integration here][2].
+
+2. Enter the full name of the repository you want to monitor. If the url for your repository is https://bitbucket.org/groupname/reponame, then enter in the Repo full name textbox
+```groupname/reponame``` .
```

Comment:
Since this is one line, it can be inline with the previous sentence:

"If the URL for your repository is https://bitbucket.org/groupname/reponame, enter `groupname/reponame` in the **Repo full name** textbox."